docs/source/ui/pages/run_management.md

```diff
@@ -189,3 +199,155 @@ Use the `Settings` tab to delete or archive the single run
+The statistics bar, below the `status` cards, displays the distribution of runs by status.
+
+#### Contributions calendar
+
+The contributions calendar represents the intensity of executed runs during the previous year. More on this read [here](../pages/home_page.html#id8).
+
+#### Activity feed
+
+This section represents the activity feed of your contributions related to the experiment. More on this read [here](../pages/home_page.html#id9).
+
+### Runs
+
+The Runs tab shows the list of runs attached to the experiment, and it also gives the ability to compare them using the available explorers.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Experiment page runs tab" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/experiment-page/experiment-page-runs-tab.png"> 
+
+### Notes
+
+The Notes tab is for attaching simple text or rich Markdown as a note to the experiment.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Experiment page notes tab" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/experiment-page/experiment-page-notes-tab.png">
+
+### Settings
+
+The Settings tab is for editing the experiment's name and description.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Experiment page settings tab" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/experiment-page/experiment-page-settings-tab.png">
+
+## Tags page
+
+### Overview
+
+The Tags functionality is intended to mark runs. A tag can be attached to runs to segment them and then quickly find them.
+
+### Create tag
+
+There are two options for creating a tag:
+
+##### First option
+
+- Go to the tags page by clicking on the "Tags" button from the left sidebar.
+
+- Click on the "Create tag" button to open the create tag form modal.
+  In this modal, there is a form that has two input fields: the first one for the tag name, and the second one for the tag comment. 
+  There are also color palettes available for selecting the tag color.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Create tag" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/tags/1.png">
+
+- Type the name for the tag. The name field is mandatory and cannot be empty for the tag creation form, and has a maximum 50 symbol limit validation.
+- Type a comment for the tag. The comment field is optional for the tag creation form and has a maximum 200 symbol limit validation.
+- Select a color for the tag from the color palette.
+- Click on the "Create" button to save the tag. After successful saving, a toast notification should appear in the top-right corner of the window to confirm the creation. 
+  There are also default selected colors for the tag available.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Create tag modal" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/tags/2.png">
+
+##### Second option
```

Comment:
I would not just write first option. I would be more specific.
Something like `Option 2: From Explorers page